client/containers/Budget/index.js

```diff
@@ -2,13 +2,13 @@ import React, { Component, PropTypes } from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
-import * as AppActions from 'actions';
+import { actions as AppActions } from 'modules/transactions';
 import BudgetGrid from 'components/BudgetGrid';
 
 @connect(
-  ({ transactions }) => ({
-    transactions: transactions.transactions,
-    categories: transactions.categories
+  ({ transactions, categories }) => ({
```

Comment:
Yes. This is correct. The previous reducer had everything grouped under a single node called `transactions`  So, we ended up with things like `transactions.transactions` and `transactions.categories`.

<img width="461" alt="screen shot 2017-03-10 at 8 51 46 am" src="https://cloud.githubusercontent.com/assets/1566467/23860033/17aac362-07dc-11e7-81f0-55cc2ead3627.png">

 Now they are all individual node in the state tree.

<img width="453" alt="screen shot 2017-03-10 at 8 45 33 am" src="https://cloud.githubusercontent.com/assets/1566467/23860059/23e19c64-07dc-11e7-8e8b-912885feba67.png">